File: 1804A-Lame King.py
for _ in range(int(input())):
    x,y=map(int,input().split())
    x=abs(x);y=abs(y)
    if x==y:
        print(abs(x+y))
    else:
        z=max(abs(x),abs(y))
        print(2*abs(z)-1)

File: 1804A-Lame King.cpp
#include<bits/stdc++.h>
#include<cmath>
#include<algorithm>
using namespace std;
int main(){
    int t,x,y;cin>>t;
    while(t--){
        cin>>x>>y;
        int x1=abs(x),y1=abs(y);
        if (x1==y1){cout<<x1+y1<<endl;}
        else{
            int z=max(x1,y1);
            cout<<2*z-1<<endl;
        }

    }

    return 0;
}